lib/driver.js

```diff
@@ -176,23 +179,32 @@ export class Driver {
             this.log("Bad addr config version: %s", pkt.version);
             return false;
         }
-        
+
         const parsed = this.handleAddrs(pkt.addrs);
         if (!parsed) return false;
 
-        const specs = parsed.map(([, s]) => s);
-        if (!await this.subscribe(specs))
-            return false;
-        
         this.addrs = new Map(parsed);
         this.topics = new Map(parsed.map(([t, s]) => [s, t]));
         this.log("Set addrs: %O", this.addrs);
         return true;
     }
 
-    async subscribe (specs) {
+    async subscribe () {
         const { handler } = this;
 
+        // Check if status is UP
```

Comment:
Do we need to handle the possibility of two copies of `subscribe` running at the same time? That would be painful...